src/CheckIfDead.php

```diff
@@ -378,15 +377,52 @@ protected function sanitizeURL( $url ) {
 			);
 		}
 		if ( isset( $parts['query'] ) ) {
-			$url .= "?".urlencode( urldecode( $parts['query'] ) );
+			$url .= "?";
+			$parts['query'] = explode( '&', $parts['query'] );
+			foreach ( $parts['query'] as $i=>$argument ) {
```

Comment:
Another coding convention failure. It should be <code>$i => $argument</code>. Note the spaces.
I would also prefer a more descriptive variable name than $i.